build/EF6Tools-VS2019-Nightly.yaml

```diff
@@ -71,8 +71,7 @@ steps:
 
 
 
-- task: ms-vseng.MicroBuildTasks.a0262b21-fb8f-46f8-bb9a-60ed560d4a87.MicroBuildLocalizationPlugin@2
-
+- task: ms-vseng.MicroBuildTasks.a0262b21-fb8f-46f8-bb9a-60ed560d4a87.MicroBuildLocalizationPlugin@3
```

Comment:
This seems to improve reliability in the Loc install step, which had been very unreliable.